packages/components/src/combobox-control/index.js

```diff
@@ -203,6 +203,11 @@ function ComboboxControl( {
 								instanceId={ instanceId }
 								ref={ inputContainer }
 								value={ isExpanded ? inputValue : currentLabel }
+								aria-label={
+									currentLabel
+										? `${ currentLabel }, ${ label }`
+										: null
```

Comment:
Is there any other default value that would be good here rather than null?

Reply:
The purpose of `null` here is to not render the `aria-label` if there's no currently selected value, because in that case, its contents are identical to the already existing label, so we don't need to override it with `aria-label`.